contrib/src/main/java/macrobase/analysis/stats/kde/ScoreEstimate.java

```diff
@@ -0,0 +1,42 @@
+package macrobase.analysis.stats.kde;
+
+import macrobase.analysis.stats.kde.kernel.Kernel;
+
+/**
+ * Class used to tracking the weight contributions of different regions of space.
+ * Stored in a priority queue and split when we need more refined estimates.
+ */
+public class ScoreEstimate {
```

Comment:
is this going to be used anywhere except in KDtree?

consider moving as inner class / helper functions in KDtree?